content/installation/node/NodeInstall.md

```diff
@@ -4,6 +4,31 @@ description: "Installing the Node.js Agent"
 tags: "NodeJS agent installation"
 -->
 
+## Prerequisites
+
+To install the Node agent, you must have installed:
+
+* A C++ compiler toolchain (e.g., Clang, GCC or MSVC) 
+
+* Python (which is needed for [Node-gyp](https://github.com/nodejs/node-gyp), Node's build tool, to function)
+
+> **Notes:** 
+ * If you can build other modules with C++ bindings like [node-sass](https://www.npmjs.com/package/node-sass) and [sqlite3](https://github.com/mapbox/node-sqlite3), you probably meet all of the requirements for Contrast.
+ * The official [Node Docker images](https://hub.docker.com/_/node/) come with all of the prerequisites installed.
+
+### Windows
+
+To install the necessary compiler toolchain on Windows, run `npm install -g --production windows-build-tools`. If you're using a Node version older than 8.0, you must use `windows-build-tools` version 3.0 (not the newest version, 4.0).
+
+### macOS
+
```

Comment:
@Avuxo It looked like all of these (Windows, macOS, etc.) are supposed be subsections under Prerequisites. If that's wrong, I can readjust the subhead levels.

Reply:
Yep, that was an oversight on my part.